### ISSUE TITLE
chapter 5.5 The Session

### ISSUE THREAD

**Issue by AnvarTurobov** [DiscoverMeteor/book#505]
Session.set('pageTitle', 'A different title'); 

returns caught ReferenceError: Session is not defined(…)

It seems session is not part of default meteor app anymore and should be added as a package.

**Comment by ghost** [comment]
Yeah, it's because the Session its not installed, To add Session to your application, run this command in your terminal:

meteor add session

https://docs.meteor.com/api/session.html
